### DISCUSSION TITLE
Setting changes via external UI

### DISCUSSION THREAD

**Discussion by apritykin** [RaspAP/raspap-webgui#814] [Q&A] [answered]
I am looking to see if it is possible to make changes to the RaspAP from a separate custom web page. What this idea comes from is sending my Raspberry to a family member and create a custom web page for someone not so "savy" on networking. Depending on the arguments the web page are, it would send the changes to the RaspAP settings.

**Comment by billz** [comment, marked as answer]
An API would facilitate this. Check out #602

**Reply by apritykin** [reply]
Fantastic! Is the endpoint the url mentioned [here](https://github.com/RaspAP/raspap-webgui/issues/602#issuecomment-789784030) ?
